sys/ztimer/Kconfig

```diff
@@ -83,12 +89,12 @@ config ZTIMER_SEC_BACKEND_TIMER
 config ZTIMER_SEC_BACKEND_RTT
     bool "RTT"
     depends on HAS_PERIPH_RTT
-    select MODULE_ZTIMER_PERIPH_RTT
+    select MODULE_ZTIMER_PERIPH_RTT if !MODULE_ZTIMER_NO_PERIPH_RTT
 
 config ZTIMER_SEC_BACKEND_RTC
     bool "RTC"
     depends on HAS_PERIPH_RTC
-    select MODULE_ZTIMER_PERIPH_RTC
+    select MODULE_ZTIMER_PERIPH_RTC if !MODULE_ZTIMER_NO_PERIPH_RTC
 
```

Comment:
maybe this is not the way it is in the dep (but as you know i am not good at reading kconfig)

Makefiles are so much easier << this is wrong